workflows/digitalai-deploy/Workflow_AWS Lambda application setup with Digital.ai Deploy.yaml

```diff
@@ -249,5 +249,5 @@ spec:
   - Application Life Cycle Management
   logo:
     type: xlrelease.TemplateLogo
-    contentType: image/png
-    file: !file "template-logo/xlr-xld-plugin.png"
+    contentType: image/svg
```

Comment:
content type should be `image/svg+xml` for SVG.

Reply:
Changes made